docs/lua-api-dimension.md

```diff
@@ -35,27 +102,93 @@ gravity = 0.5
 
 Default value is `1`.
 
-### `id`
+#### `horizontal_sprint_mod`
 
-ID of the dimension. **Mandatory field.**
+Horizontal sprint speed multiplier.
 
 Example:
 ```lua
-id = "dimension_nether"
+horizontal_sprint_mod = 1.5
 ```
 
-IDs are usually of the form `mod:dimension` but the `mod:` prefix is prepended automatically so it's not needed.
+Default value is `1.5`.
 
-### `name`
+#### `is_sneak_always_mod`
 
-Label of the dimension. **Mandatory field.**
+Sprint speed multiplier.
 
 Example:
 ```lua
-name = "Nether"
+is_sneak_always_mod = false
 ```
 
-This label is the name that will appear everywhere in the game.
+Default value is `false`.
+
+#### `jump_antigravity`
+
+The gravity of the dimension is decreased by this amount while holding space prior to the peak of your jump.
```

Comment:
- you can remove `of the dimension`
- `while holding the jump key prior to the peak of the jump.`